From 6aad0284013ee20a4d49c19e5d992191511a9be2 Mon Sep 17 00:00:00 2001
From: Brandon Brnich <b-brnich@ti.com>
Date: Wed, 12 Jul 2023 15:50:04 -0500
Subject: [PATCH 136/143] TEMP: media: img: vxe-vxd: decoder: Disable CMA for
 capture buffers

CMA alloc issues occurring for larger resolution
videos. Enable MMU to handle capture buffer management.

Signed-off-by: Brandon Brnich <b-brnich@ti.com>
---
 drivers/media/platform/img/vxe-vxd/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/img/vxe-vxd/Makefile b/drivers/media/platform/img/vxe-vxd/Makefile
index 3270153b22cd..a01d90320d1b 100644
--- a/drivers/media/platform/img/vxe-vxd/Makefile
+++ b/drivers/media/platform/img/vxe-vxd/Makefile
@@ -44,7 +44,7 @@ ERROR_RECOVERY_SIMULATION ?=n
 # This config enables allocation of capture buffers from
 # dma contiguous memory.
 # If unsure, say Y
-CAPTURE_CONTIG_ALLOC ?=y
+CAPTURE_CONTIG_ALLOC ?=n
 
 vxd-dec-y += common/img_mem_man.o \
 	common/img_mem_unified.o \
-- 
2.41.0

